src/neuroconv/baseextractorinterface.py

```diff
@@ -2,11 +2,11 @@
 from abc import ABC
 from typing import Optional
 
-from .basedatainterface import BaseDataInterface
+from .basetemporalalignmentinterface import BaseTemporalAlignmentInterface
 from .tools import get_package
 
 
-class BaseExtractorInterface(BaseDataInterface, ABC):
```

Comment:
I also think it is too early to make this contract. Are we sure that all the objects from spikeinterface and roiextractors will be alignable with our methods? If not, then we are overcommiting. I think we should separate this.

Reply:
These are already done

Audio is the final interface to refactor to use these methods

Reply:
OK. I also could not think why someone that is not us would like to inherit from this class so this makes sense. I still have the lingering feeling that adding this capablities to the interfaces should be done through composition but this probably does not matter. Just leaving the thought.

Reply:
What do you mean by composition? It's not too late to change anything yet